lib/src/request.rs

```diff
@@ -12,7 +12,9 @@ mod edge_event_buffer;
 pub use self::edge_event_buffer::EdgeEventBuffer;
 
 use crate::line::{self, EdgeEvent, Offset, Value, Values};
-use crate::{AbiVersion, Error, Result, UapiCall};
+use crate::{Error, Result, UapiCall};
+#[cfg(feature = "uapi_v1")]
+use crate::AbiVersion;
```

Comment:
^^^ AbiVersion is only used if the "uapi_v1" feature is defined, either by itself or with both defined.